### PR TITLE
Plugins: Enable catalog management link to gcom

### PR DESCRIPTION
**Pull request by wbrowne**
In order to support use of the Plugin catalog with Hosted Grafana environments, we can now callout to grafana.com where plugins can be managed externally.

This PR adds a new config `plugin_admin_remote_manage_enabled`, which will set `Install`, `Uninstall`, and `Update` buttons to link to https://grafana.com/grafana/plugins/{pluginID} instead of installing directly on the filesystem.

For example:

image

.. where clicking will open a new tab to [https://grafana.com/grafana/plugins/grafana-timestream-datasource](https://grafana.com/grafana/plugins/grafana-timestream-datasource)

image

**NOTE**: I have intentionally omitted the new config from `configuration.md` as this config is only relevant for HG which will be manually controlled. Users won't ever need/want to manually this manually

### CODE REVIEW

File: conf/defaults.ini [hunk 1]
```diff
@@ -885,6 +885,7 @@ app_tls_skip_verify_insecure = false
 allow_loading_unsigned_plugins =
 # Enable or disable installing plugins directly from within Grafana.
 plugin_admin_enabled = false
+plugin_admin_remote_manage_enabled = false
```
Comment (marefr) [inline conf/defaults.ini:888]: Nit. For the external user management, https://grafana.com/docs/grafana/latest/administration/configuration/#external-user-management, we use external. Here we  use remote. Do we want to make this consistent?

**Review by marefr — APPROVED** [review]
LGTM, but added a bit comment/question

File: conf/defaults.ini (hunk 1 above)
Comment (wbrowne) [reply conf/defaults.ini:888]: Good point. I'm easy either way. How about you @bergquist?

**Review by jackw — APPROVED** [review]
LGTM! 🚀

File: docs/sources/administration/configuration.md [hunk 2]
```diff
@@ -1473,12 +1473,10 @@ Enter a comma-separated list of plugin identifiers to identify plugins that are
 
 ### plugin_admin_enabled
 
-
 Available to Grafana administrators only, the plugin admin app is set to `false` by default. Set it to `true` to enable the app.
 
 For more information, refer to [Plugin catalog]({{< relref "../plugins/catalog.md" >}}).
```
Comment (oddlittlebird) [inline docs/sources/administration/configuration.md:1479]: Don't forget to add **plugin_admin_remote_manage_enabled here.

**Review by oddlittlebird — CHANGES_REQUESTED** [review]

File: docs/sources/administration/configuration.md (hunk 2 above)
Comment (wbrowne) [reply docs/sources/administration/configuration.md:1479]: See note in the PR description
>NOTE: I have intentionally omitted the new config from configuration.md as this config is only relevant for HG which will be manually controlled. Users won't ever need/want to manually this manually

Comment (oddlittlebird) [reply docs/sources/administration/configuration.md:1479]: If the option is visible to all users in the .ini docs, then we need to define it, even if all we say is, "Only applicable to Grafana Cloud users."

**Review by oddlittlebird — APPROVED** [review]
LGTM

File: conf/defaults.ini (hunk 1 above)
Comment (bergquist) [reply conf/defaults.ini:888]: No strong opinion but consistency is always nice :)

**Comment by bergquist** [comment]
:+1: how do I install the plugin catalog if I want to try this out?

File: conf/defaults.ini (hunk 1 above)
Comment (wbrowne) [reply conf/defaults.ini]: Done 😄

**Review by marefr — APPROVED** [review]
LGTM

**Pull request merged**